README.md

```diff
@@ -252,7 +252,7 @@ A curated list of awesome Java frameworks, libraries and software.
 *Tools which handle the distribution of applications in native formats.*
 
 * [Bintray](https://bintray.com/) - Version control for binaries which handles the publishing. Can also be used with Maven or Gradle and has a free plan for open-source software or several business plans.
-* [Capsule](http://capsule.io) - Simple and powerful packaging and deployment. A fat JAR on steroids or a "Docker for Java" that supports JVM-optimized containers.
+* [Capsule](http://www.capsule.io/) - Simple and powerful packaging and deployment. A fat JAR on steroids or a "Docker for Java" that supports JVM-optimized containers.
```

Comment:
It follows the http redirect, so what happened is
- http://www.gradle.org/ redirects to http://gradle.org/ but
- http://capsule.io redirects to http://www.capsule.io/

It is based on what the company wants its url to be :smile: does it make sense?